autoload/fzf/vim.vim

```diff
@@ -405,7 +405,7 @@ function! s:git_status_sink(lines) abort
 endfunction
 
 function! fzf#vim#gitfiles(args, ...)
-  let root = systemlist('git rev-parse --show-toplevel')[0]
+  let root = split(system('git rev-parse --show-toplevel'), '(\n|\r\n)')[0]
```

Comment:
Can you change `'(\n|\r\n)'` to simply `'\n'`? The regex of vim is eccentric and requires you to escape parentheses, and considering that fzf does not run on Windows, I think `'\n'` should suffice.